nnf/__init__.py

```diff
@@ -998,6 +998,53 @@ def pair(node: NNF) -> NNF:
 
         return pair(sentence)
 
+    def project(self: T_NNF, vars: 't.FrozenSet[Name]') -> 'NNF':
+        """Dual of `forget`: will forget all of the variables not given"""
```

Comment:
If you write it like this it turns into a link to the method:
```suggestion
        """Dual of :meth:`forget`: will forget all of the variables not given"""
```
Note that if  you want verbatim text like markdown's backticks you need double backticks, i.e.
```
``forget``
```